Assets/_Application/Layers/LayerTypes/CartesianTiles/CartesianTileLayerGameObject.cs

```diff
@@ -184,15 +187,27 @@ public override void ApplyStyling()
                 foreach (var (_, feature) in LayerFeatures)
                 {
                     Symbolizer symbolizer = GetStyling(feature);
-                    CartesianTileLayerStyler.Apply(symbolizer, feature);
-                    if (feature.Geometry is not Material material) continue;
 
-                    Color? color = symbolizer.GetFillColor();
-                    if (color.HasValue)
+                    if (feature.Geometry is ObjectMappingItem item)
                     {
-                        if (int.TryParse(feature.Attributes[CartesianTileLayerStyler.MaterialIndexIdentifier], out var materialIndex))
-                        {                            
-                            binaryMeshLayer.DefaultMaterialList[materialIndex].color = color.Value;
+                        bool? visiblity = symbolizer.GetVisibility();
+                        if (visiblity.HasValue)
+                        {
+                            string id = feature.Attributes[CartesianTileLayerStyler.VisibilityAttributeIdentifier];
+                            var visibilityColor = visiblity == true ? symbolizer.GetFillColor() ?? Color.white : Color.clear;
```

Comment:
kan deze regel misschien iets worden uitgeschreven, het is een beetje onduidelijk zo